src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -102,11 +108,15 @@ private void validateMinSize() {
         }
     }
 
-    private void updateByUpStation(Section section) {
-        sections.stream()
+    private boolean updateByUpStation(Section section) {
+        Optional<Section> optional = sections.stream()
             .filter(it -> it.isUpStationOfSection(section))
-            .findFirst()
-            .ifPresent(it -> it.updateUpStationBySection(section));
+            .findFirst();
+        optional.ifPresent(it -> it.updateUpStationBySection(section));
+        if (optional.isPresent()) {
+            return true;
+        }
+        return false;
```

Comment:
inline 처리가 가능할 것 같아요 :)

```suggestion
        return optional.isPresent();
```